src/styles/blocks/card.scss

```
@@ -0,0 +1,131 @@
+$w-cont: #fff;
+$color-b: #060b35;
+$cont-color: #616070;
+$blue-cont: #00acdc;
+
+html {
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-style: normal;
+}
+
+body {
```

reset default margin for p and h2 here
```suggestion
body,
p,
h2 {
```